scripts/covid_tracking_project/historic_us_data/README.md

```diff
@@ -0,0 +1,17 @@
+# Importing The COVID Tracking Project's Historic US Data
+
+This directory contains artifacts required for importing
+The COVID Tracking Project's Historic US Data
+into Data Commons, along with scripts used to generate these artifacts.
+
+To generate `COVIDTracking_US.tmcf` and `COVIDTracking_US.csv`, run:
+
+```bash
+python3 preprocess_csv.py
+```
+
+#### Note:
+
+`COVIDTracking_US_StatisticalVariables.mcf` is actually the same as `COVIDTracking_States_StatisticalVariables.mcf` in /historic_state_data/ folder, so we don't need to check it here.
```

Comment:
Could we just remove the file?

Reply:
Deleted

Reply:
I meant delete the COVIDTracking_US_StatisticalVariables.mcf not the README 😬 Sorry for the confusion

Reply:
Hahaha, no worry~ Recovered